src/components/navigation/navigation-item/navigation-item.component.html

```diff
@@ -0,0 +1,7 @@
+<a *ngIf="link" (click)="toggle(); $event.stopPropagation()" [class.has-arrow]="this.children.length > 0" [class.no-arrow]="this.indentWithoutArrow()" [routerLink]="link">

```

Comment:
You don't toggle if you have a link - NavigationEnd should handle the expansion of parents.

Reply:
toggle() removed from anchor element with router link.